src/components/router.tsx

```diff
@@ -15,4 +15,4 @@ createMessage
   .get("/", (ctx) => ctx.html(<Home />))
   .get("/create", (ctx) => ctx.html(<CreateMessageTop />))
   .get("/select", (ctx) => ctx.html(<CreateMessageSelect />))
-  .get("/check", (ctx) => ctx.html(<CreateMessageCheck />));
+  .get("/check", (ctx) => ctx.html(CreateMessageCheck(ctx)));
```

Comment:
```suggestion
  .get("/check", (ctx) => {
    const id = ctx.req.query("id") || "";
    return ctx.html(<CreateMessageCheck id={id} />);
  });
```
昨日試してたら、こんな感じじゃないと僕の方は動かなかったです

確認してもらえると 🙏

Reply:
修正しましたー
[141cb03](https://github.com/jigintern/online-2024-e/pull/47/commits/141cb038c52d16f38309ab1041095791c7cc0836)